public/app/features/alerting/unified/hooks/useFilteredRules.ts

```diff
@@ -217,6 +225,10 @@ const reduceGroups = (filterState: RulesFilter) => {
         matchesFilterFor.ruleType = true;
       }
 
+      if ('plugins' in matchesFilterFor && filterState.plugins === 'hide') {
+        matchesFilterFor.plugins = !isPluginProvidedRule(rule);
```

Comment:
should not be the opposite? 🤔
`matchesFilterFor.plugins = isPluginProvidedRule(rule);`